substrate/primitives/core/src/ecdsa.rs

```diff
@@ -438,7 +479,14 @@ impl TraitPair for Pair {
 impl Pair {
 	/// Get the seed for this key.
 	pub fn seed(&self) -> Seed {
-		self.secret.secret_bytes()
+		#[cfg(feature = "std")]
+		{
+			self.secret.secret_bytes()
+		}
+		#[cfg(not(feature = "std"))]
+		{
+			self.secret.to_bytes().into()
+		}
```

Comment:
```suggestion
		#[cfg(feature = "std")]
		self.secret.secret_bytes()
		#[cfg(not(feature = "std"))]
		self.secret.to_bytes().into()
```

Reply:
Unfortunately we need to keey `{...}` here, as what you proposed does not compile:
```
error: expected `;`, found `#`
   --> substrate/primitives/core/src/ecdsa.rs:483:30
    |
483 |             self.secret.secret_bytes()
    |                                       ^ help: add `;` here
484 |         #[cfg(not(feature = "std"))]
    |         - unexpected token
```

This is why I kept `{}` above (for consistency).